docs/guide/third_party.rst

```diff
@@ -0,0 +1,54 @@
+Third-party plugins
+===================
+This page explains how ``napari-matplotlib`` can be used within third party plugins.
+
+``napari-matplotlib`` provides a ready-to-go widget with a Matplotlib toolbar and figure to third party plugin developers
+This widget is customised to match the theme of the main napari window.
+
+The widget can be found at `napari_matplotlib.base.NapariMPLWidget`.
+This class inherits from ``qtpy.QtWidgets.QWidget``.
+
+It is recommended to use `~napari_matplotlib.base.NapariMPLWidget` inside a new widget, adding it to the layout.
```

Comment:
```suggestion
We recommend you use `~napari_matplotlib.base.NapariMPLWidget` inside a new widget, adding it to the layout.
```
...?
Since it's a clear actor doing an action (_we recommend_).

Reply:
I'm reluctant to use "We", but I've changed this to slightly less clunky language

Reply:
But... **we** do. One day I'll convince you.